### DISCUSSION TITLE
Can't open HDR AVIF

### DISCUSSION THREAD

**Discussion by radogado** [ImageMagick/ImageMagick#6855] [Help]
### ImageMagick version

7.1.1-21

### Operating system, version and so on

macOS 13.6.1

### Description

Hi, I looked everywhere and can't understand why it fails to convert a 10-bit HDR AVIF Rec. 2020 (which opens fine in Chrome)
**magick DSC06260.avif a.jpeg**
doesn't work and produces
**magick: Invalid input: Invalid box size: Box size (0 bytes) smaller than header size (8 bytes) (2.101) `DSC06260.avif' @ error/heic.c/IsHEIFSuccess/138.**
Any idea what to do? Thanks!

**Comment by snibgo** [comment]
I suggest you link to `DSC06260.avif`, or zip it and upload it here, so we can test it.

**Reply by radogado** [reply]
Attached below. Thanks!

**Comment by radogado** [comment]
[DSC06260.avif.zip](https://github.com/ImageMagick/ImageMagick/files/13301148/DSC06260.avif.zip)

**Comment by snibgo** [comment]
I can't reproduce the problem. On Windows 11, IM v7.1.1-20, there is no problem reading your file.

What does "`magick -list format`" say about AVIF? For example, I get:

```
C:\web\im>%IMG7%magick -list format |grep -i avif
     AVIF  rw+   AV1 Image File Format (1.16.2)
```
Can you read other AVIF files?

**Reply by radogado** [reply]
`magick -list format |grep -I avif`
`AVIF  HEIC      rw+   AV1 Image File Format (1.17.0)`

I can't read this 10-bit image: https://gregbenzphotography.com/assets/hdr/gallery/wide/Prana-hdr.avif
I can read this 8-bit one: https://res.cloudinary.com/ddxwdqwkr/image/upload/q_60/v1632079580/smashing-articles/photo-1533371452382-d45a9da51ad9_1.avif

Will try also on Windows later. Thanks!

**Reply by radogado** [reply]
It works on Windows:
`ImageMagick 7.1.1-21 Q16-HDRI arm64`
`magick Prana-hdr.avif a.jpeg`

**Comment by snibgo** [comment]
> I can't read this 10-bit image: https://gregbenzphotography.com/assets/hdr/gallery/wide/Prana-hdr.avif

Again, I can read it with no problem. Sorry.

**Comment by radogado** [comment]
When downloading for Windows, I chose a version with HDR enabled. Does this mean it's disabled on Mac (brew)?

**Comment by radogado** [comment]
On Windows, when resizing the HDR image, the result isn't HDR. How to fix that? Thanks!
